### ISSUE TITLE
[winesteam]:Data path for SteamApp not found

### ISSUE THREAD

**Issue by VortexAcherontic** [lutris/lutris#1346]
Hey there,

I was today trying to improve some winesteam installers I wrote in the past.
But I noticed that lutris is, again, not installing winesteam games.

I tested 3 different games and expected always the same behavior.
I found a closed issue on this but that one was about non default locations.
Becuase I work with a default fresh new lutris config folder everything should be in it's default locations.
(The wine steam runner was installed before I ran the install scripts with the prefix: esync-staging-pba-3.18-x86_64)

How ever in the terminal I get the following warning and nothing happens ever again:

`WARNING  <date and time> [winesteam]:Data path for SteamApp <appid> not found.`

I tested the following installers which where working in the past:
https://lutris.net/games/elite-dangerous/
https://lutris.net/games/warhammer-vermintide-2/
https://lutris.net/games/the-turing-test/

**Comment by VortexAcherontic** [comment]
I also noticed that lutris is not installing wine with a fresh config folder so I ran:

```
export WINEPREFIX="/home/<user>/.local/share/lutris/runners/winesteam/prefix64/"                                                                
export WINEPATH="/home/<user>/.local/share/lutris/runners/wine/tkg-3.19-x86_64/bin/wine" 
cd "/home/<user>/.local/share/lutris/runners/winesteam/prefix64/drive_c/Program Files (x86)/Steam/"     
wine Steam.exe    
```

**Comment by sigmaSd** [comment]
to install winesteam, lutris uses this `subprocess.Popen(command, env=self.get_env())` it will fail if you have a env that it doesn't understand, for example I had `TERM=xterm-256color` it did not like it, it only worked after I exported `TERM=xterm`

**Comment by VortexAcherontic** [comment]
Oh, thanks, that worked! :D

**Comment by sigmaSd** [comment]
@VortexAcherontic , Would you keep the issue open? we need to remember to handle that error^^

**Comment by VortexAcherontic** [comment]
@sigmaSd no problem =)

**Comment by strycore** [comment]
this is fixed by not setting prefer system libs to true by default so fixed in 0.5.0
